_includes/sections/browser-webrtc.html

```diff
@@ -38,10 +38,21 @@ <h3>How to disable WebRTC in Firefox?</h3>
 <a class="btn btn-primary" target="_blank" rel="noopener noreferrer" href="https://ipleak.net">Test your Browser again</a>
 </p>
 
-<h3>How to fix the WebRTC Leak in Google Chrome?</h3>
+<h3>How to disable WebRTC in Safari?</h3>
+
+<p>Safari is far stricter with WebRTC than other major browsers, and it does not leak your IP address in its default configuration. If you'd like, you can follow these steps just to double-check your browser:</p>
+
+<ol class="long-string-list">
+  <li>Choose "Preferences" under the Safari menu in your menu bar.</li>
+  <li>Select the Advanced tab and check the "Show Develop menu in menu bar" box.</li>
+  <li>Exit Preferences and open the Develop menu in your menu bar.</li>
+  <li>In the drop-down menu, open the "WebRTC" submenu and ensure "Enable Legacy WebRTC API" is unchecked. If it's grayed out, even better.</li>
+</ol>
+
+<h3>How to disable WebRTC in Google Chrome?</h3>
 
 <p>WebRTC cannot be fully disabled in Chrome; however, it is possible to change its routing settings (and prevent leaks) using an extension. Two open-source solutions include <a href="https://chrome.google.com/webstore/detail/webrtc-leak-prevent/eiadekoaikejlgdbkbdfeijglgfdalml">WebRTC Leak Prevent</a> (options may need to be changed depending on the scenario), and <a href="https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm">uBlock Origin</a> (select "Prevent WebRTC from leaking local IP addresses" in Settings).</p>
 
 <h3>What about other browsers?</h3>
 
-<p>Chrome on iOS, Internet Explorer and Safari does not implement WebRTC yet. <a href="#browser"><i class="fas fa-link"></i> But we recommend using Firefox on all devices.</a></p>
+<p>Chrome on iOS and Internet Explorer does not implement WebRTC yet. <a href="#browser"><i class="fas fa-link"></i> But we recommend using Firefox on all devices.</a></p>
```

Comment:
```suggestion
<p>Chrome on iOS and Internet Explorer do not implement WebRTC yet. <a href="#browser"><i class="fas fa-link"></i> But we recommend using Firefox on all devices.</a></p>
```

> Chrome on iOS

[citation needed]? This may be the same as Safari now (in that WebRTC is supported, because I believe it is supported on Safari on iOS) -- Alternatively *if* Chrome on iOS doesn't implement WebRTC, we should check if Firefox on iOS and Safari on iOS does implement it in the first place, because they would be worth adding to this list if not. I'll double-check later...

Reply:
IIRC I saw the same info somewhere...